### PR TITLE
chore: improve GitHub templates

### PR DESCRIPTION
**Pull request by u11d-michal-miler**
# Description

Improve GitHub templates:

* Pull Request
* issue
* commit

# Type of change

- [ ] build: Build related changes (eg: npm related/ adding external dependencies)
- [x] chore: A code change that external user won't see (eg: change to .gitignore file or .prettierrc file)
- [ ] feat: A new feature
- [ ] fix: A bug fix
- [ ] docs: Documentation related changes
- [ ] refactor: A code that neither fix bug nor adds a feature. (eg: You can use this when there is semantic changes like renaming a variable/ function name)
- [ ] perf: A code that improves performance
- [ ] test: Adding new test or making changes to existing test

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/technical_improvement.md [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
+
 A clear and concise description of the technical improvement you are suggesting.
 
-**Current implementation**
+## Current implementation
+
 If applicable, provide a brief explanation of the current implementation related to the improvement.
 
-**Benefits**
+## Benefits
```
Comment (u11d-bartlomiej-galezowski) [inline .github/ISSUE_TEMPLATE/technical_improvement.md:15]: IMO that also could be optional.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,23 +1,26 @@
-# Description
+## Description
 
-Please include a summary of the changes (use imperative, present tense (eg: use "add" instead of "added" or "adds")).
+<!--
+Please include a summary of the changes.
+Use imperative, present tense (i.e.: use "add" instead of "added" or "adds").
+-->
```
Comment (u11d-bartlomiej-galezowski) [inline .github/pull_request_template.md:6]: Make all "descriptions" as comment.

Comment (u11d-michal-miler) [reply .github/pull_request_template.md:6]: done

File: .github/ISSUE_TEMPLATE/technical_improvement.md (hunk 1 above)
Comment (u11d-michal-miler) [reply .github/ISSUE_TEMPLATE/technical_improvement.md]: done

**Review by u11d-bartlomiej-galezowski — APPROVED** [review]

**Pull request merged**